app/src/main/java/com/github/se/orator/ui/overview/FeedbackScreen.kt

```diff
@@ -207,15 +207,17 @@ fun FeedbackScreen(
                             .padding(top = AppDimensions.paddingMedium)
                             .border(
                                 width = AppDimensions.borderStrokeWidth,
-                                color = AppColors.buttonBorderColor,
+                                color = MaterialTheme.colorScheme.outline.copy(alpha = 0.5f),
```

Comment:
Don't you think it would also be better to use a colorScheme color here ?

Reply:
In fact, but it is already done.

Reply:
In fact, but it is already done.